### PR TITLE
speed up the algorinthm by 20000%

### PR DESCRIPTION
**Pull request by nakoyasha**
this removes all of the bloatware microservices that were used to promote woke content, this removes it and boosts the true patriots's posts

this pull is approved by cement
she chjose to eat cement when she was 6

### CODE REVIEW

**Comment by Foxlass** [comment]
This should be approved

**Review by angrymouse — COMMENTED** [review]
LGTM

File: .gitignore
Comment (angrymouse) [inline .gitignore:1]: lgtm

**Review by angrymouse — COMMENTED** [review]
Ready for merge!

**Comment by royeradames** [comment]
You are just dealing all the files. Lol

**Comment by nakoyasha** [comment]
> You are just dealing all the files. Lol

Nuhuh i implemented a better solution in the 2nd commit

**Comment by jjjakey** [comment]
Given condition 1 appears to full fill the requirements of condition 2, I'd recommend nesting the first condition inside the second for improved efficiency.

**Comment by pa-t** [comment]
can you add test coverage?

**Comment by mateusmlo** [comment]
Will this fix also work on my brand new Tesla and make it run faster?

**Comment by dromzeh** [comment]
LGTM

**Comment by tas-pro** [comment]
you may want to change the language of choice

**Comment by SooshMeow** [comment]
LGTM

**Comment by GAMBLING-MAN** [comment]
nice! no bloatware remaining! merge immediately devs!

**Comment by paisenpai** [comment]
LGTM ! :)

**Comment by altruong** [comment]
brilliant refactor!

**Review by erikmork1 — COMMENTED** [review]
LGTM

**Comment by Edwardsoen** [comment]
LGTM

**Comment by vfrascello** [comment]
Merge

**Comment by hexaquarks** [comment]
LGTM, ship it.

File: main.lua [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        local is_a_rightwing_dumbass = user:is_a_rightwing_dumbass()
+        local is_elon = user:is_elon()
+
+        if (is_elon) then
+            user:shadow_ban()
+        elseif (is_a_rightwing_dumbass) then
+            user:shadowed_shadow_very_shadow_ban()
+        else
+            -- WIN!
+            user:boost(20000)
```
Comment (PROxZIMA) [inline main.lua:14]: Please add proof of claim.
Performance comparison is necessary here.
The code looks appealing though.

**Review by PROxZIMA — COMMENTED** [review]
Please see the following requirements

**Comment by gottagofaster236** [comment]
LGTM, merge immediately

**Comment by darkpixlz** [comment]
lgtm, ready for prod

File: main.lua [hunk 2]
```diff
@@ -0,0 +1,17 @@
+local algo = twitter.algo
+
+while (algo:active()) do
+    for _, user in pairs(algo.users) do
+        local is_a_rightwing_dumbass = user:is_a_rightwing_dumbass()
+        local is_elon = user:is_elon()
+
+        if (is_elon) then
```
Comment (darkpixlz) [inline main.lua:8]: Remove () for performance. Besides that, lgtm

**Review by pyxfluff — COMMENTED** [review]
Lgtm

**Pull request closed without merging**